example-data/peerscout-api-request2.json

```diff
@@ -1,7 +1,7 @@
 {
     "manuscript_id":"67890",
     "tracking_number":"67890_xyZ",
-    "revision_number":"1",
+    "revision_number":1,
     "abstract":"Second Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
     "title":"Second Lorem Ipsum",
     "include_reviewing_editors_id":[
```

Comment:
might be worth to make those slightly more explicit that they are author suggested.
e.g. `author_suggested_to_include_reviewing_editors_id` (although it is a bit long)

Reply:
The alternative could be to put those fields inside an `author_suggestion` field, what do you think?

Reply:
that makes sense